pretyping/tacred.ml

```diff
@@ -47,9 +47,15 @@ let error_not_evaluable r =
     (str "Cannot coerce" ++ spc () ++ Nametab.pr_global_env Id.Set.empty r ++
      spc () ++ str "to an evaluable reference.")
 
+let is_primitive env c =
+  let cb = lookup_constant c env in
+  match cb.Declarations.const_body with
+  | Declarations.Primitive _ -> true
+  | _ -> false
```

Comment:
This function should live in environ.ml

Reply:
I'm not sure what happened to this function, but it seems no longer there.

Reply:
Of course, I'm blind. Now moved to `Environ`.